docker/mock-lpa-store/mock-lpa-store-config.yaml

```diff
@@ -1,5 +1,6 @@
 plugin: openapi
 specFile: https://github.com/ministryofjustice/opg-data-lpa-store/raw/main/docs/openapi/openapi.yaml
-
 validation:
   request: false
+response:
+  scriptFile: lpa-store.groovy
```

Comment:
the JS wasn't compiling for some reason, I doubt we'll make it complicated enough that using groovy becomes a problem